templates/footer.php

```diff
@@ -0,0 +1,7 @@
+<script src="js/jquery.js"></script>
+<script src="js/bootstrap.bundle.js"></script>
+<script src="js/main.js"></script>
```

Comment:
You should remove the main.js line from here

Reply:
yes 
done